packages/flame/test/util/expect_double.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:test/test.dart';
+
+void expectDouble(
+  double d1,
+  double d2, {
+  double epsilon = 0.0001,
```

Comment:
The epsilon is much smaller here than it was for `closeTo` which will probably give us some more flakiness in the effects tests.